.rubocop.yml

```diff
@@ -96,7 +96,7 @@ Metrics/MethodLength:
     - "app/controllers/v0/institutions_controller.rb"
     - "lib/roo_helper/loader.rb"
     - "lib/seed_utils.rb"
-
+    - "app/controllers/v0/institution_programs_controller.rb"
```

Comment:
Is there a way to break up the method instead of disabling this rubopcop check?

Reply:
Hi @LindseySaari, I just wanted to follow up. I was able to merge in the separate Gemfile specific PR yesterday and was also able to break up this method which resolved any rubocop/code climate issues for this PR.